packages/data-point/lib/reducer-types/reducer-list/factory.js

```diff
@@ -78,7 +77,7 @@ module.exports.parse = parse
  * @param {Array} source
  * @return {reducer}
  */
-function create (createReducer, source) {
+function create (createReducer, source = []) {
```

Comment:
why do we need a default parameter here?

Reply:
There's a test case asserting that calling factory.create with only one argument should result in a ReducerList with an empty set of reducers. But this test broke when removed the calls to _.compact

Reply:
We could also take this default case out and just change the test

Reply:
Yeah let's just remove that test instead of adding the default param. Thanks